### PR TITLE
Changelog for 0.9 release

### PR DESCRIPTION
**Pull request by tomwilkie**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
+## Release 0.9.0
+
+New features:
+- Add basic Kubernetes views for pods and services
+  [#441](https://github.com/weaveworks/scope/pull/441)
+- Add containers-by-hostname view
+  [#545](https://github.com/weaveworks/scope/pull/545)
+- Make ```scope launch``` work from remote hosts, with an appropriately defined DOCKER_HOST
```
Comment (peterbourgon) [inline CHANGELOG.md]: Just FYI single-ticks are fine inline

**Comment by 2opremio** [comment]
I have tested this branch with the ECS demo and I am missing edges between the http server and the data producers:

screen shot 2015-10-26 at 2 11 34 pm


here's the report: https://gist.github.com/2opremio/455b0c294e603748cd5e

And here are the demo machines in case @tomwilkie wants to have a look at them:

```
  http://ec2-52-30-203-246.eu-west-1.compute.amazonaws.com
  http://ec2-52-30-243-163.eu-west-1.compute.amazonaws.com
  http://ec2-52-30-1-231.eu-west-1.compute.amazonaws.com
```

Also, the CLI now prints an error when checking the docker version: https://github.com/weaveworks/scope/issues/588

**Comment by tomwilkie** [comment]
If you tested with 1.2, this regression is caused by that: https://github.com/weaveworks/weave/issues/1577

I suggest you disable FDP and test again.

**Comment by 2opremio** [comment]
Yep, I was running FDP. As for the second (admittedly minor) bug, I have a PR for this branch: #591

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+- Add `scope command` for printing the Docker commands used to run Scope
+  [#553](https://github.com/weaveworks/scope/pull/553)
+- Include some basic documentation on how to run Scope
+  [#572](https://github.com/weaveworks/scope/pull/572)
+- Warn if the users tries to run Scope on Docker versions <1.5.0
+  [#557](https://github.com/weaveworks/scope/pull/557)
+- Add support for loading the Scope UI from https endpoings
+  [#572](https://github.com/weaveworks/scope/pull/572)
+
+Big fixes:
```
Comment (paulbellamy) [inline CHANGELOG.md]: Do you mean "Bug fixes" ? or "Big fixes"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+  [#524](https://github.com/weaveworks/scope/pull/524)
+- Increase DNS poll frequency such that Scope clusters up more quickly
+  [#524](https://github.com/weaveworks/scope/pull/524)
+- Add `scope command` for printing the Docker commands used to run Scope
+  [#553](https://github.com/weaveworks/scope/pull/553)
+- Include some basic documentation on how to run Scope
+  [#572](https://github.com/weaveworks/scope/pull/572)
+- Warn if the users tries to run Scope on Docker versions <1.5.0
+  [#557](https://github.com/weaveworks/scope/pull/557)
+- Add support for loading the Scope UI from https endpoings
```
Comment (paulbellamy) [inline CHANGELOG.md]: May want to add ssl support for apps on port 443.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+  [#572](https://github.com/weaveworks/scope/pull/572)
+
+Big fixes:
+- Correctly track short-lived connections from the internet
+  [#493](https://github.com/weaveworks/scope/pull/493)
+- Fix a corner case where short-lived connections between containers are incorrectly attributed
+  [#577](https://github.com/weaveworks/scope/pull/577)
+- Ensure service credentials are sent when doing intial probe<->app handshake
```
Comment (paulbellamy) [inline CHANGELOG.md]: intial

**Comment by paulbellamy** [comment]
I can't seem to replicate the fix for https://github.com/weaveworks/scope/pull/493

**Comment by peterbourgon** [comment]
While I understand there are some remaining issues with conntrack, my issues are resolved; LGTM.

**Pull request merged**